src/app/pages/FrontPage/index.jsx

```diff
@@ -58,13 +59,20 @@ export const StyledFrontPageDiv = styled.div`
   }
 `;
 
+const GreyBox = styled.div`
+  background: #ccc;
+  height: 5rem;
+  padding-top: 1rem;
+`;
+
 const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
   const {
     product,
     serviceLocalizedName,
     translations,
     frontPageTitle,
   } = useContext(ServiceContext);
+  const { enabled } = useToggles('ads');
```

Comment:
Can I make a small request here:

 `const { enabled: adsEnabled } = useToggles('ads');` - just for readability